vdr/vdr.go

```diff
@@ -76,14 +79,15 @@ func (r *VDR) Configure() error {
 
 	r.configOnce.Do(func() {
 		if r.networkAmbassador == nil {
-			r.networkAmbassador = NewAmbassador(r.network)
+			r.networkAmbassador = NewAmbassador(r.network, r.store, crypto.Instance())
```

Comment:
it should forward the cryptoClient from the vdr struct, not create a new instance.